src/main/java/com/lorepo/icplayer/client/module/text/TextPresenter.java

```diff
@@ -1948,4 +1986,73 @@ private void addiOSClass() {
 			view.addIOSClass();
 		}
 	}
+
+	private void sendTimerEvent() {
+		long time = timerBase / 1000;
+		if (this.startTime != null) {
+			Date newTime = new Date();
+			time = (timerBase + (newTime.getTime() - this.startTime.getTime())) / 1000;
```

Comment:
Propozycja:
`time += (newTime.getTime() - this.startTime.getTime()) / 1000;`

Reply:
Powiedzmy, że aktualny timerBase to 3600ms, an różnica w czasie między newTime a startTime to 1600ms, w sumie 5200ms. Według zaproponowanej przez Ciebie metody zostałoby to przeliczone na 4s (3600ms->3s, 1600ms->1s).

Reply:
Masz rację. Cofam propozycję